plivo/resources/token.py

```diff
@@ -0,0 +1,23 @@
+import json
+from plivo.base import (ListResponseObject, PlivoResource,PlivoResourceInterface)
+from plivo.utils import to_param_dict
+from plivo.utils.validators import *
+
+class Token(PlivoResourceInterface):
+    @validate_args(
+        iss = [required(of_type(six.text_type))],
+        sub = [optional(of_type(six.text_type))],
+        nbf = [optional(of_type(six.text_type))],
+        exp = [optional(of_type(six.text_type))],
+        incoming_allowed = [optional(of_type(bool))],
+        outgoing_allowed = [optional(of_type(bool))],
+        app = [optional(of_type(six.text_type))]
+    )
+
+    def create( self, iss, sub = None, nbf = None, exp = None, incoming_allowed = None, outgoing_allowed = None, app = None):    
```

Comment:
outgoing_allowed and incoming_allowed should be replaced with [incoming_allow and outgoing_allow]